wolfhsm/wh_server.h

```diff
@@ -42,54 +73,65 @@ typedef struct whServerContext_t whServerContext;
 #include "wolfhsm/wh_nvm.h"
 #include "wolfhsm/wh_message_customcb.h"
 
-#ifndef WOLFHSM_NO_CRYPTO
+#ifndef WOLFHSM_CFG_NO_CRYPTO
 #include "wolfssl/wolfcrypt/settings.h"
+#include "wolfssl/wolfcrypt/types.h"
 #include "wolfssl/wolfcrypt/random.h"
 #include "wolfssl/wolfcrypt/rsa.h"
 #include "wolfssl/wolfcrypt/ecc.h"
 #include "wolfssl/wolfcrypt/curve25519.h"
 #include "wolfssl/wolfcrypt/cryptocb.h"
-#endif /* WOLFHSM_NO_CRYPTO */
+#endif /* WOLFHSM_CFG_NO_CRYPTO */
 
-#ifdef WOLFHSM_SHE_EXTENSION
+#ifdef WOLFHSM_CFG_SHE_EXTENSION
 #include "wolfhsm/wh_she_common.h"
 #include "wolfhsm/wh_server_she.h"
 #endif
 
 /** Default server resource configurations */
-/* Number of RAM keys */
-#ifndef WOLFHSM_NUM_RAMKEYS
-#define WOLFHSM_NUM_RAMKEYS  8
+/* Reported version string */
+#ifndef WOLFHSM_CFG_INFOVERSION
+#define WOLFHSM_CFG_INFOVERSION "01.01.01"
+#endif
+
+/* Reported build identifier string */
+#ifndef WOLFHSM_CFG_INFOBUILD
+#define WOLFHSM_CFG_INFOBUILD "12345678"
 #endif
 
+
 /* Number of NVM objects in the directory */
-#ifndef WOLFHSM_NUM_NVMOBJECTS
-#define WOLFHSM_NUM_NVMOBJECTS  32
+#ifndef WOLFHSM_CFG_NVM_OBJECT_COUNT
+#define WOLFHSM_CFG_NVM_OBJECT_COUNT 32
 #endif
 
-/* Size in bytes of key cache buffer  */
-#ifndef WOLFHSM_KEYCACHE_BUFSIZE
-#define WOLFHSM_KEYCACHE_BUFSIZE 1200
+/* Number of RAM keys */
+#ifndef WOLFHSM_CFG_SERVER_KEYCACHE_COUNT
+#define WOLFHSM_CFG_SERVER_KEYCACHE_COUNT  8
+#endif
+
+/* Size in bytes of each key cache buffer  */
+#ifndef WOLFHSM_CFG_SERVER_KEYCACHE_BUFSIZE
+#define WOLFHSM_CFG_SERVER_KEYCACHE_BUFSIZE 1200
 #endif
 
 /* Custom request shared defs */
-#ifndef WH_CUSTOM_CB_NUM_CALLBACKS
-#define WH_CUSTOM_CB_NUM_CALLBACKS 8
+#ifndef WOLFHSM_CFG_SERVER_CUSTOMCB_COUNT
+#define WOLFHSM_CFG_SERVER_CUSTOMCB_COUNT 8
 #endif
 
 /* DMA translation allow entries */
-#ifndef WH_DMA_ADDR_ALLOWLIST_COUNT
-#define WH_DMA_ADDR_ALLOWLIST_COUNT 10
+#ifndef WOLFHSM_CFG_SERVER_DMAADDR_COUNT
+#define WOLFHSM_CFG_SERVER_DMAADDR_COUNT 10
```

Comment:
small opinion/nit, feel free to disregard: I kind of like keeping the "allow" word in these defines, because it really indicates the number of addresses in the allowlist.